backend/models/office_hours/oh_section.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseModel
+
+__authors__ = ["Sadie Amato"]
+__copyright__ = "Copyright 2024"
+__license__ = "MIT"
+
+
+class OfficeHoursSection(BaseModel):
```

Comment:
@KrisJordan can't remember, do we have a convention for the names of association entities? Wondering if naming such that "Event" is included in the name would be better since this is the association table btwn `oh__event` and `academics__section`, or if such a name would be too long.

Reply:
Our office hours section table isn't an association table between `oh__event `and `academics__section`; it's a one-to-many with the `academics__section`. It just looks like one since there are only a few fields!